lib/streamlit/MediaFileManager.py

```diff
@@ -41,7 +41,7 @@ def _get_file_id(data, mimetype=None):
     filehash.update(bytes(mimetype.encode("utf-8")))
     return filehash.hexdigest()
 
-    #return hashlib.sha224(bytes(mimetype.encode("utf-8")) + data).hexdigest()
+    # return hashlib.sha224(bytes(mimetype.encode("utf-8")) + data).hexdigest()
```

Comment:
I don't know how this snuck back in here, but you can remove it.  It's code I thought I removed.

Reply:
(Or I will remove it in my next PR if you don't feel like it.)

Reply:
Removed!